src/s3ql/statfs.py

```diff
@@ -31,6 +33,10 @@ def parse_args(args):
 
     parser.add_argument("--raw", action="store_true", default=False,
                           help="Do not pretty-print numbers")
+    parser.add_argument("--prometheus_exporter", action="store_true", default=False,
```

Comment:
Please move all the Prometheus stuff into a separate pull request, otherwise it's very hard to review.

Reply:
Okey, I will create another branch and move prometheus things to other PR to check after work with gdrive implementation